extensions/binding/org.eclipse.smarthome.binding.dmx/META-INF/MANIFEST.MF

```diff
@@ -9,9 +10,12 @@ Bundle-Vendor: Eclipse.org/SmartHome
 Bundle-Version: 0.10.0.qualifier
 Export-Package: 
  org.eclipse.smarthome.binding.dmx,
- org.eclipse.smarthome.binding.dmx.handler
+ org.eclipse.smarthome.binding.dmx.handler,
+ org.eclipse.smarthome.binding.dmx.action
 Import-Package: 
  org.eclipse.jdt.annotation;resolution:=optional,
+ org.eclipse.smarthome.automation,

```

Comment:
This line should not be required (and it actually now makes the DMX binding dependent on the experimental rule engine, which is not good).